website/docs/configuration/outputs.html.md

```diff
@@ -98,10 +98,53 @@ output "db_password" {
 }
 ```
 
-Setting an output value in the root module as sensitive prevents Terraform
-from showing its value in the list of outputs at the end of `terraform apply`.
-It might still be shown in the CLI output for other reasons, like if the
```

Comment:
The spirit of this section, referencing <0.14 behavior, has been migrated to the **Note** below